src/bin/hamcore/languages.txt

```diff
@@ -5,4 +5,5 @@
 0 ja Japanese 日本語 1041 ja,jp,sjis,shift_jis,euc
 1 en English English 1033 en,us,c
 2 cn Simplified_Chinese 简体中文	 2052,4100,1028,3076,5124 zh,cn,tw,hkg,mac,sg,chi
+3 ko Korean 한국어	 949 ko,kr,euc_kr,cp949,euckr

```

Comment:
Could you also please use the number "4" because another PR will take the number "3"